dashboard/lib/user_multi_auth_helper.rb

```diff
@@ -17,7 +17,7 @@ def migrate_to_multi_auth
 
     unless sponsored?
       self.primary_authentication_option =
-        if %w(clever facebook google_oauth2 powerschool windowslive).include? provider
+        if AuthenticationOption::OAUTH_CREDENTIAL_TYPES.include? provider
```

Comment:
Note: Although we've added credential types Quikcamps, The School Project, Twitter, to this list, they're still not supported by the migrate function due to the "Migration not implemented" check against user provider type, above.  We'll unblock those after we have tests for them (if we choose to support them).

This check against `provider` might seem like it goes against what I said about not converting for `user.provider`, but in this part of the migration the user provider string _must_ be an exact match for the credential type string, so I don't feel bad about it.